src/main/java/com/redhat/darcy/webdriver/internal/DelegatingWebContext.java

```diff
@@ -330,27 +332,48 @@ public Alert alert() {
             FindsByAttribute context = (FindsByAttribute) contextForType(type);
 
             return context.findByAttribute(type, attribute, value);
-        } catch (ClassCastException cce) {
+        } catch (ClassCastException e) {
             throw unsupportedLocatorForType("HTML attribute, " + attribute + "'", type);
```

Comment:
Tiny tiny nit (hey, it counts though?): maybe don't need the extra single quote after attribute since it looks like `unsupportedLocatorForType` already adds one.